Formula/crystal.rb

```diff
@@ -95,12 +103,12 @@ def install
     # Setup the same path the wrapper script would, but just for building shards.
     # NOTE: it seems that the installed crystal in bin/"crystal" can be used while
     #       building the formula. Otherwise this ad-hoc setup could be avoided.
-    embedded_crystal_path=`"#{buildpath/".build/crystal"}" env CRYSTAL_PATH`.strip
-    ENV["CRYSTAL_PATH"] = "#{embedded_crystal_path}:#{buildpath/"src"}"
+    embedded_crystal_path = Utils.safe_popen_read(buildpath/".build/crystal", "env", "CRYSTAL_PATH").strip
+    ENV["CRYSTAL_PATH"] = "#{embedded_crystal_path}:#{buildpath}/src"
 
     # Install shards
     resource("shards").stage do
-      system "make", "bin/shards", "CRYSTAL=#{buildpath/"bin/crystal"}",
+      system "make", "bin/shards", "CRYSTAL=#{buildpath}/bin/crystal",
```

Comment:
That's a wrapper script that exists in the source tree. It basically configures the compiler binary to use the associated stdlib source, because it's not installed yet and the baked in path wouldn't resolve.

Reply:
Great; thanks for the clarification. Any feedback/objections before I merge this?

Reply:
Using the source tree wrapper script means the previous setting of `ENV["CRYSTAL_PATH"]` should be unnecessary. It can probably be removed. The path should be configured correctly from the source tree wrapper.